docs/index.md

```diff
@@ -0,0 +1,95 @@
+# Planet SDK for Python
+
+[![Build Status](https://travis-ci.org/planetlabs/planet-client-python.svg?branch=master)](https://travis-ci.org/planetlabs/planet-client-python)
+
+[Planet](https://planet.com) Software Development Kit (SDK) for Python.
+
+## Quick Start
```

Comment:
this is out of scope for this initial PR, but just curious - is there a way we can use a single source-of-truth for this content instead of duplicating things on README and docs/index.md both?

Reply:
100% agree the duplication is clunky and prone to errors. Looking into the issue raised around this in mkdocs (https://github.com/mkdocs/mkdocs/issues/321#issuecomment-70106615), it looks like our options are linking to `docs/README.md` a stub root `README.md` or duplicating things.

Reply:
Lets continue this in #277